Header Header
POI_Objects[] poi_objects
bool poi_present
bool obstacle_present
string obstacle_label
string turn_direction
float64 forward_distance